docs/auction-house/definition.md

```diff
@@ -36,39 +36,58 @@ AuctionHouse is a completely separate program than the storefront Auction progra
 
 :::
 
 #### Requires Sign Off Feature
-This feature allows a marketplace to restrict which NFTs get sold on their platform. It is useful for more centralized marketplaces or a marketplace that has order matching algorithms that the user has allowed them to use (enabled by listing the NFT at the price of 0). 
+
+This feature allows a marketplace to restrict which NFTs get sold on their platform. It is useful for more centralized marketplaces or a marketplace that has order matching algorithms that the user has allowed them to use (enabled by listing the NFT at the price of 0).
 
 Marketplaces who want to stay decentralized and not require signoff may restrict what their user interfaces show via other means, but behind the scenes, someone can still list an NFT on your Auction House. You may in this scenario build Allow Lists using merkele trees or chose to restrict what your UI shows via other means.
 
 ### Any SPL Token
-AuctionHouse allows you to accept any SPL token as the tender that the buyer deposits into their Buyer Escrow in order to accomplish a sale. 
+
+AuctionHouse allows you to accept any SPL token as the tender that the buyer deposits into their Buyer Escrow in order to accomplish a sale.
 
 Now that you know what the AuctionHouse is, take a look at our [Getting Started](/auction-house/getting_started) guide.
 
 ### Auction House Receipts
+
 To aid transaction tracking, Auction House supports the generation of receipts for listings, bids, and sales. To generate these receipts, the receipt printing function should be called immediately after the corresponding transaction (`PrintListingReceipt`, `PrintBidReceipt`, and `PrintPurchaseReceipt`). Additionally, the `CancelListingReceipt` and `CancelBidReceipt` instructions should be called in the case of canceled listing add bids. Calling these two instructions will fill the `canceled_at` fields of the `ListingReceipt` and `BidReceipt` accounts.
 
 :::info
 
 While the receipts can be retrieved using the standard getProgramAccounts data flow, the official recommendation is to use Solana's [AccountsDB](https://docs.solana.com/developing/plugins/geyser-plugins) plug-in to index and track the generated receipts.
+:::
+
+### Partial Order Fulfillment
+
+A seller can create a sell order of `FungibleAssets` with a quanity greater than 1. The buyer can then create a buy order of said assets that is less than the total amount of the sell order. In order for `ExeculeSale` to succeed, the buy order must have been created with both a `partial_order_size` and a `partial_order_price`. `partial_order_price` must match the division of total tokens in the full order and the total buy price times the `partial_order_size`. `partial_order_size` must not be greater than the total amount of tokens in the original sell order. If there is no partial order needing to take place, `partital_order_price` and `partial_order_size` can be passed in as `None`.
```

Comment:
Small typo w/ quantity/quanity, otherwise perfect!

Reply:
shiiiiiii

Reply:
typo fixed and pushed @blockiosaurus